src/Discord.Net.WebSocket/Entities/Guilds/SocketGuild.cs

```diff
@@ -504,7 +504,7 @@ internal AudioInStream GetAudioStream(ulong userId)
         {
             return _audioClient?.GetInputStream(userId);
         }
-        internal async Task<IAudioClient> ConnectAudioAsync(ulong channelId, bool selfDeaf, bool selfMute, Action<IAudioClient> configAction)
+        internal async Task<IAudioClient> ConnectAudioAsync(ulong channelId, bool selfDeaf, bool selfMute, Action<IAudioClient> configAction, bool external)
         {
             selfDeaf = false;
```

Comment:
A little unrelated, but @foxbot is there any reason we explicitly set these to false here?

Reply:
Okay, remove these assignments so that it can be passed up from the parent call.